oxidation/libparsec/crates/platform_device_loader/src/native/local_device.rs

```diff
@@ -7,7 +7,7 @@ use libparsec_client_types::{DeviceFile, LocalDevice, LocalDeviceResult};
 use crate::load_device_file;
 use crate::load_device_with_password_core;
 
-pub fn load_device_with_password_from_path(
+pub async fn load_device_with_password_from_path(
```

Comment:
The call to `load_device_file` should be made async

Reply:
Done.